docs/make.jl

```diff
@@ -0,0 +1,22 @@
+using SymbolServer
+using Documenter
+
+makedocs(;
+    modules=[SymbolServer],
+    authors="Julia VSCode",
+    repo="https://github.com/julia-vscode/SymbolServer.jl/blob/{commit}{path}#L{line}",
+    sitename="SymbolServer.jl",
+    format=Documenter.HTML(;
+        prettyurls = prettyurls = get(ENV, "CI", nothing) == "true",
+        # canonical="https://julia-vscode.github.io/SymbolServer.jl",
+        # assets=String[],
+    ),
```

Comment:
Presumably we can remove the whole `format` part?

Reply:
This helps to build documents offline and have urls embedded into html instead of having seperate folders.